src/Widgets/DataFormNumberWidget/DataFormNumberWidgetComponent.tsx

```diff
@@ -63,7 +63,7 @@ provideComponent(DataFormNumberWidget, ({ widget }) => {
         className="form-control"
         defaultValue={defaultValue}
         id={id}
-        name={attributeName}
+        name={attributeName ?? ''}
```

Comment:
* [ ] FYI maybe `{attributeName ?? undefined}` expresses the state more clearly. Not sure how exactly form submissions behave with nameless vs. empty-named fields.

Reply:
I think both will fail due to

https://github.com/Scrivito/scrivito-portal-app/blob/b5b853dde539cd5a87e857034d4ea51c7c703bc0/src/Widgets/DataFormContainerWidget/DataFormContainerWidgetComponent.tsx#L113-L114